src/components/Chart/ReferenceAreaTick.tsx

```diff
@@ -22,6 +22,8 @@ export function ReferenceAreaTick(props: ReferenceTickProps) {
   const theme = useTheme()
 
   const nameLines = splitIntoLines(ref.name, MAX_CHARS_PER_LINE)
+  const description = ref.description
+  const textYOffset = ref.textYOffset ?? 0
```

Comment:
Just arguing a little more, I still think this is a good approach because it removes the need for the user to calculate the offset needed to place the name between the rectangles

Reply:
If the style, font or font-size changes, then a constant is not enough for customization.

Reply:
Fair enough